src/Microsoft.DotNet.XHarness.iOS.Shared/Microsoft.DotNet.XHarness.iOS.Shared.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <IsPackable>true</IsPackable>
-    <TargetFrameworks>netstandard2.1;netcoreapp3.1</TargetFrameworks>

```

Comment:
why leave in 2.1 if everything else is up to 5... is this used by a 2.1 project?

Reply:
iOS.Shared is consumed in .NET 4.8.2 in `xamarin/xamarin-macios`